platform-tests/src/test/java/org/junit/platform/engine/discovery/FilePositionTests.java

```diff
@@ -31,12 +32,7 @@
  * @since 1.0
  */
 @DisplayName("FilePosition unit tests")
-class FilePositionTests extends AbstractTestSourceTests {
-
-	@Override
-	Stream<FilePosition> createSerializableInstances() {
-		return Stream.of(FilePosition.from(42, 99));
-	}
```

Comment:
`AbstractTestSourceTests` is only accessible from `org.junit.platform.engine.support.descriptor`, so I though it preferable to keep that that way. Should this particular override and the functionality it covered be reproduced here? Or should I not yet have moved this tests class at this time?